srcs/Channel.cpp

```diff
@@ -54,7 +54,9 @@ void Channel::removeUser(const Client *const client) {
 	userMap.erase(nickname);
 }
 
-const std::string &Channel::getName() const { return _name; }
+const std::string &Channel::getTopic() const { return _topic; }
```

Comment:
is it necessary, check and remove if not

Reply:
yes it is necessary